### PR TITLE
$NSFWtype[...]

### PR DESCRIPTION
**Pull request by Darkgoatie**
## Type
- [x] Functions: \ $NSFWtype[Image-url]

Dependencies | Url | Size
- NSFWJS  | [Github Link](https://github.com/infinitered/nsfwjs) | 2.64 MB
 
Discord Tag: `Darkgoatie#6381`

### First
Added $NSFWtype[Image-url]
Returns one of these:
Name     | Description
----------|-------------------------------------------
Drawing | safe for work drawings (including anime)
Hentai    | hentai and pornographic drawings
Neutral  | safe for work neutral images
Porn | pornographic images, sexual acts
Sexy | sexually explicit images, not pornography

### Second
Added $includesCount[Phrase;Word to search]
Returns how many words in `phrase` incule `word to search`
Example usage:
```js
$includesCount[I love apples, because apples are sweet!;apple] // Returns 2
$includesCount[Pasta kraken cake bomb;a] // Returns 3
```

### CODE REVIEW

**Comment by USERSATOSHI** [comment]
@Darkgoatie u forgot parser.js

add the function in parser.js too

**Comment by Darkgoatie** [comment]
Done, updated parser.js

**Comment by Darkgoatie** [comment]
Added $includesCount, also fixed parser.js

> @Darkgoatie u forgot parser.js
> 
> add the function in parser.js too

**Comment by DiscordBond** [comment]
Suggesting the names $imageNSFW and $containsAmount ^^

**Comment by Darkgoatie** [comment]
> Suggesting the names $imageNSFW and $containsAmount ^^

Okay, i'll rename them eventually. Will this be accepted?

**Comment by USERSATOSHI** [comment]
> > Suggesting the names $imageNSFW and $containsAmount ^^
> 
> Okay, i'll rename them eventually. Will this be accepted?
> 

@Darkgoatie that's decided by the managers 
we don't know if they like it or not ¯\_(ツ)_/¯

**Comment by Darkgoatie** [comment]
Oh okay, i'll wait then :D

File: package/functions/funcs/NSFWtype.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+	if (err) return d.error(err)
+const img = inside
+  
+nsfwjs.load()
+  .then(function (model) {
+    return {
+        code: code.replaceLast(`$NSFWtype${inside}`, model.classify(inside).deleteBrackets())
+    }
+  })
+
```
Comment (Kino7916) [inline package/functions/funcs/NSFWtype.js:16]: No variables with name "nsfwjs" was defined.

**Comment by Kino7916** [comment]
It is best to name the function as `$classifyImage`, based on how will it return types.

**Comment by Darkgoatie** [comment]
Okay, i'll decide abt the name soon

**Comment by Rubenennj** [comment]
fyi hentai and porn are nsfw words, this function makes no sense

**Pull request closed without merging**